src/Behat/Behat/Context/Snippet/Generator/ContextSnippetGenerator.php

```diff
@@ -28,72 +30,53 @@
  */
 final class ContextSnippetGenerator implements SnippetGenerator
 {
-    /**
```

Comment:
Please keep this comment as it provides info about the array shape for this property

Reply:
Although I think `string[string]` is not correct - is that even valid? Shouldn't it be `array<string,array<string,string>>`?